erpnext/templates/pages/product_search.py

```diff
@@ -42,5 +43,17 @@ def get_product_list(search=None, start=0, limit=12):
 		"today": nowdate()
 	}, as_dict=1)
 
+	for item in data:
+		product_info = get_product_info_for_website(item.item_code)
```

Comment:
this is duplicate code. Please convert to a function!

Reply:
@rmehta Thanks for the review, function added in PR #13335